woocommerce/woocommerce-gateway-stripe
# feat: UPE payment method and manual capture

# Changes proposed in this Pull Request:

Helps with https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1874 - splitting into multiple PRs

Disable the payment method checkbox when "Manual capture" is enabled.
Show a notice on UPE payment methods when "Manual capture" is enabled.

![2021-10-05 12 52 53](https://user-images.githubusercontent.com/273592/136076692-66a9fa3f-8a8b-457a-8dd6-39818598f802.gif)

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- With "Manual capture" disabled, the notice is not displayed
- With "Manual capture" enabled, the notice is displayed

**PLEASE NOTE:** In order to disable/enable the "Manual capture" setting, please manually edit the default value of the `useManualCapture` hook in `client/settings/payments-and-transactions-section/data-mock.js`
